ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor/ControlChannel.hs

```diff
@@ -53,31 +55,41 @@ instance Show peerAddr
 
 
 
--- | A Server control channel which instantiates 'handle'.
+-- | A Server control channel which instantiates to 'NewConnection' and 'Handle'.
+--
+-- It allows to pass 'STM' transactions which will resolve to 'NewConnection'.
+-- Server's monitoring thread is the consumer of these messages; there are two
+-- producers: accept loop and connection handler for outbound connections.
 --
 type ServerControlChannel (muxMode :: MuxMode) peerAddr versionData bytes m a b =
```

Comment:
What about renaming this type to `InboundGovernorControlChannel` and the other one to `OutboundGovernorChannel`.

Reply:
That's actually better!

Reply:
The `Ouroboros.Network.InboundGovernor.InboundGovernorInfoChannel` should be moved here.

Reply:
Actually this file should be deleted because I refactored it to InformationChannel.hs